src/shared/components/modal.jsx

```diff
@@ -129,7 +126,9 @@ export default class Modal extends React.Component {
       <Backdrop
         onClick={e => {
           e.stopPropagation();
-          this.props.onClose();
+          if (this.props.onClose) {
```

Comment:
if this is a required prop, we should not have to guard against it being missing

Reply:
That's a good point. I'm not sure it should be required, though. I don't use it in *file-modal.jsx*. It may need to be removed entirely as you say elsewhere.